src/Compilers/VisualBasic/Portable/VisualBasicExtensions.vb

```diff
@@ -164,14 +164,19 @@ End Namespace
 
 Namespace Microsoft.CodeAnalysis.VisualBasic
     Public Module VisualBasicExtensions
+        ''' <summary>
+        ''' Determines if the given raw kind value belongs to the Visual Basic <see cref="SyntaxKind"/> enumeration.
+        ''' </summary>
+        ''' <param name="rawKind">The raw value to test.</param>
+        ''' <returns><c>true</c> when the raw value belongs to the Visual Basic syntax kind; <c>false</c> otherwise.</returns>
```

Comment:
```suggestion
        ''' <returns><see langword="true"/> when the raw value belongs to the Visual Basic syntax kind; otherwise, <see langword="false"/>.</returns>
```